modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/prepare/IgnitePlanner.java

```diff
@@ -133,6 +133,12 @@ public class IgnitePlanner implements Planner, RelOptTable.ViewExpander {
 
     private RelOptCluster cluster;
 
+    static {
+        // Preload some classes to reduce first planning time.
```

Comment:
In the fact visible time of first planning will be the same. 
I see tow options here:
1) Move loading and initialization classes on earlier stage, during node startup. Due to java could do classpoading in parallel it shouldn't significant increase time of node start, but our very first parsing will be visible faster. For tests which could instantiate some of services himself we could keep initialization in two places.
2) Just fix a coomet to show real life )

Reply:
Thanks, I expanded the comment because we have tests that use the planner directly, without starting nodes, services, etc.